### PR TITLE
[metric] Modify object store memory chart to use object_store_memory

### PR DESCRIPTION
**Pull request by rickyyx**
Signed-off-by: rickyyx <rickyx@anyscale.com>

## Why are these changes needed?
image

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by rickyyx** [comment]
Unfortunately - I wasn't able to get a dashboard with this change working. Running into issues building ray from source on workspaces, and using wheels....

**Comment by rickyyx** [comment]
Another thing that I need to do is to set `RAY_event_stats_print_interval_ms=100`, otherwise default value is 1min, which means we only get recording once per minute.

**Review by rkooo567 — DISMISSED** [review]

**Comment by rkooo567** [comment]
Btw, I can't see this from the Ray dashboard. Is it expected? Should we update some config to make that happen?

**Comment by rkooo567** [comment]
Oh i think you need to update metrics.tsx?

**Comment by rickyyx** [comment]
Hmmm - I thought I updated the grafana dashboard in-place so it will use the same url. But I might be wrong.

**Comment by ericl** [comment]
Shall we fix the polling interval before merging? Otherwise, this graph will refresh much slower than the other ones.

**Comment by rickyyx** [comment]
I am overwriting the current scheduler object store memory - should I add a new one actually?

File: dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json [hunk 1]
```diff
@@ -319,26 +319,26 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "ray_resources{Name=\"object_store_memory\",State=\"USED\"} / 1e9",
+          "expr": "sum(ray_object_store_memory / 1e6) by (Location)",
```
Comment (ericl) [inline dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json:322]: Can we keep to GiB here for consistency with other units?

File: dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json [hunk 2]
```diff
@@ -319,26 +319,26 @@
...
           "interval": "",
           "legendFormat": "Object Store: {{instance}}",
           "queryType": "randomWalk",
           "refId": "A"
         },
         {
           "exemplar": true,
-          "expr": "sum(ray_resources{Name=\"object_store_memory\"} / 1e9)",
+          "expr": "",
```
Comment (ericl) [inline dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json:330]: Hmm we still want the MAX query. Can you put this back?

Comment (rickyyx) [reply dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json:330]: Yeah - I agree.

Comment (rickyyx) [reply dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json:330]: Kind of messed up the json.

**Comment by ericl** [comment]
Sounds good. Btw, could you post a screenshot with the new rendering for
metric PRs? I think that's good practice for UI changes.

**Review by ericl — APPROVED** [review]

**Pull request merged**